attributes/default.rb

```diff
@@ -57,8 +57,9 @@
 default['cfncluster']['ganglia']['web_url'] = 'https://github.com/ganglia/ganglia-web/archive/3.7.2.tar.gz'
 # NVIDIA
 default['cfncluster']['nvidia']['enabled'] = 'no'
-default['cfncluster']['nvidia']['driver_url'] = 'http://download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run'
-default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux'
+# Is there a non-US domain?
```

Comment:
remove

Reply:
actually is there a non us domain? :D

Reply:
I couldn't find it :(

Reply:
for cn
* https://cn.download.nvidia.com/tesla/418.87/NVIDIA-Linux-x86_64-418.87.00.run
* https://developer.download.nvidia.cn/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_418.87.00_linux.run